docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,43 @@
+# How to add self-signed certificate to trusted certificates list
+
+## Overview
+
+For secure connectivity to work you need to add CA certificate to the trusted certificate list repository.
+For the most part the store will be filled with certificates from your tls/ssl provider, but when you want to use self-signed certificates you may need to update your local certificate store.
+Below are instructions on how to add new CA certificate and update the certificate store.
+
+> Please note: Provided instructions are for supported OSes and may not apply to the flavour you are running, if you need help with other OS please consult appropriate documentation.
+
+## Ubuntu (and Raspberry Pi OS)
+
+> If you do not have the ca-certificates package installed on your system, install it with your package manager. For Ubuntu or Raspberry Pi OS that would be to invoke following command:
```

Comment:
You don't need "For Ubuntu or Raspberry Pi OS" here as it's under the heading for those OSes.